### PR TITLE
add temporary notes data

### PR DESCRIPTION
**Pull request by Letushev**
Generate dump mock data to use until we finish storage setup.

### CODE REVIEW

File: src/notes.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
+[
+  {
+    "title": "Meeting Agenda",
+    "description": "Discuss quarterly sales projections and marketing strategies with the team.",
+    "createdAt": "2024-05-09T08:00:00Z",
+    "editedAt": "2024-05-09T08:00:00Z"
```
Comment (tetianaman) [inline src/notes.json]:
```
interface Note {
    id: string;
    title: string;
    description: string;
    createdDate: Date;
    modifiedDate: Date;
    deletedDate?: Date;
    isPinned: boolean;
}
```

WDYT?

Comment (ynesteruk) [inline src/notes.json:6]: We also need to know if note is pinned/removed

Comment (ynesteruk) [reply src/notes.json]: IMO: description -> contents

**Comment by ynesteruk** [comment]
I would think about general solution to use it. A board template with mocked data provider, but it depends on how store would be implemented

File: src/notes.json (hunk 1 above)
Comment (tetianaman) [reply src/notes.json]: smth like this in progress [Data management API - PR](https://github.com/wixplosives/NotesApp/pull/6/files)
GPT generated. 
I'm testing it ...

Comment (Letushev) [reply src/notes.json]: Added `id`, `isPinned` and optional `deletedAt`

**Review by tetianaman — APPROVED** [review]

**Pull request closed without merging**